Automattic/wp-calypso
# Uses itemLinkMatches for Me paths.

#### Changes proposed in this Pull Request

* Instead of static mapping of a path to a Sidebar Item, this PR uses `itemLinkMatches` for all paths in **Me** section. This approach is more dynamic and already used in **My Sites** section #50318
* Removes feature flags which are now true in all envs.

#### Testing instructions



* Spin up the calypso live
* Go to `/me`
* Check that all Sidebar Items are selected correctly when clicked
* Use the secondary navigation of some pages (eg Purchases) and make sure that the Sidebar Item is still selected

Before | After
-------|------
![](https://cln.sh/f7HuQa+) | ![](https://cln.sh/CI0rlQ+)



Fixes #49631
